# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/dav1d/.gitlab-ci.yml
# Commit: a5108ffbd311
# Full Hash: a5108ffbd31101e7cb72dabc271d92027f953c3c
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-04-20 09:35:22
# Regressor Bug: 1540830
# File Overlap Count: 3
# Description:
#   Bug 1540830 - Update dav1d from upstream to 1f7a7e8. r=TD-Linux
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D28200
# ==============================================================================

diff -r 98adabf295d0 -r a5108ffbd311 third_party/dav1d/.gitlab-ci.yml
--- a/third_party/dav1d/.gitlab-ci.yml	Fri Apr 19 20:49:36 2019 +0000
+++ b/third_party/dav1d/.gitlab-ci.yml	Fri Apr 19 20:36:10 2019 +0000
@@ -12,6 +12,12 @@
     script:
         - git grep -n -P "\t|\r| $" -- . ':(exclude)*/compat/*' && exit 1
         - git grep -n -i "david" -- . ':(exclude)THANKS.md' ':(exclude).gitlab-ci.yml' && exit 1
+        - for i in $(git ls-files -- . ':(exclude)*/compat/*'); do
+              if [ -n "$(tail -c 1 "$i")" ]; then
+                  echo "No newline at end of $i";
+                  exit 1;
+              fi;
+          done
         - git remote rm upstream 2> /dev/null || true
         - git remote add upstream https://code.videolan.org/videolan/dav1d.git
         - git fetch -q upstream master